### PR TITLE
store/tikv:move MaxTxnTimeUse from kv to store/tikv

### PR DESCRIPTION
**Pull request by AndreMouche**
Signed-off-by: AndreMouche

### What problem does this PR solve?
This PR moves `MaxTxnTimeUse` from `kv` to `store/tikv` package.
Part of https://github.com/pingcap/tidb/issues/22513

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: store/tikv/oracle/oracle.go [hunk 1]
```diff
@@ -141,3 +141,12 @@ func GoTimeToTS(t time.Time) uint64 {
 	ts := (t.UnixNano() / int64(time.Millisecond)) << physicalShiftBits
 	return uint64(ts)
 }
+
+// MaxTxnTimeUse is the max time a Txn may use (in ms) from its begin to commit.
+// We use it to abort the transaction to guarantee GC worker will not influence it.
+const MaxTxnTimeUse = 24 * 60 * 60 * 1000
```
Comment (disksing) [inline store/tikv/oracle/oracle.go:147]: Seems not very suitable to move the const to here. Maybe we can pass it to `GoTimeToLowerLimitStartTS` as another parameter.

Comment (jackysp) [reply store/tikv/oracle/oracle.go:147]: Agree.

**Review by JmPotato — APPROVED** [review]

File: domain/infosync/info.go [hunk 2]
```diff
@@ -556,7 +557,7 @@ func (is *InfoSyncer) ReportMinStartTS(store kv.Storage) {
 		return
 	}
 	now := time.Unix(0, oracle.ExtractPhysical(currentVer.Ver)*1e6)
-	startTSLowerLimit := oracle.GoTimeToTS(now.Add(-time.Duration(kv.MaxTxnTimeUse) * time.Millisecond))
+	startTSLowerLimit := oracle.GoTimeToLowerLimitStartTS(now,tikv.MaxTxnTimeUse)
```
Comment (sticnarf) [inline domain/infosync/info.go:560]: space after `,`

**Review by sticnarf — APPROVED** [review]

**Pull request merged**